sphinx_rtd_theme/layout.html

```diff
@@ -9,9 +9,7 @@
 {%- set lang_attr = 'en' if language == None else (language | replace('_', '-')) %}
 {%- set sphinx_writer = 'writer-html5' if html5_doctype else 'writer-html4' -%}
 
-{# Build sphinx_version_info tuple from sphinx_version string in pure Jinja #}
-{%- set (_ver_major, _ver_minor, _ver_bugfix) = sphinx_version.split('.') | map('int') -%}
-{%- set sphinx_version_info = (_ver_major, _ver_minor, _ver_bugfix) -%}
+{%- set sphinx_vn = parse_version(sphinx_version) -%}
```

Comment:
I renamed it because it wasn't the same thing as before. We could certainly build a ``sphinx_version_info`` tuple for compatibility, if that is a thing we need to be careful of.